deployment/examples/docker-compose.yaml

```diff
@@ -15,14 +15,22 @@ services:
       POSTGRES_PASSWORD: obs
       POSTGRES_DB: obs
     volumes:
-    - ./data/postgres/data:/var/lib/postgresql/data
+      - ./data/postgres/data:/var/lib/postgresql/data
+    labels:
+      - traefik.enable=false
+    networks:
+      - backend
+    # Uncomment, when you want to run osm2pgsql
+    #  - gateway
+    #ports:
+    #  - '127.0.0.1:5432:5432'
 
   portal:
     image: openbikesensor-portal
     build:
       context: ./source
     volumes:
-      - ./data/api-data:/data
+      - ./data/api-data:/opt/obs/data
```

Comment:
same as above - are you sure? I haven't needed this change in either in my migrated production nor in my from-scratch dev-setup. Also in the from scratch setup tracks properly arrive in the volume. The python code seems to indicate that the data is looked for in ``/opt/obs/api/../../../data`` in the default container layout (which would mean ``/data``).

Reply:
My setup is only working like this. As **DATA_DIR**, I have the default configured as in the [config.py.example](https://github.com/openbikesensor/portal/blob/main/api/config.py.example). I had the issue, that the upload or the worker failed, since it was not able to find the tracks, thus I had to change it.

I'll try to reproduce it on a new server instance after x-mas.